### DISCUSSION TITLE
frameness?

### DISCUSSION THREAD

**Discussion by ahdee** [stjude/CICERO#64] [Q&A] [answered]
Hi is there a way to have Cicero output the frame predictions? Such as, in.frame, out.of.frame or undetermine? 

edit: I notice 2 colums, frame and sv_frame but can't make out how to utilized these to determine the frame. 

thanks!

**Comment by adthrasher** [comment, marked as answer]
@mnedmonson - Can you describe these two columns from the frame check? 

I believe there is a numeric code [0,3] with blank also a possibility. 
0 - event is not in frame
1 - event is in-frame
2 - geneB portion contains canonical coding start site
3 - possible 5' UTR fusion in geneB

**Reply by mnedmonson** [reply]
The "frame" field is a comma-delimited list of frame-calling codes, each index in the list also refers to other comma-delimited fields afterwards in the output, e.g. sv_refseqA and sv_refseqB tell you which pair of isoforms is being evaluated.  For the frame codes, a a blank value indicates the frame check could not be performed -- there are a number of reasons for this, for example if the contig doesn't contain coding sequence for both genes, or if the UCSC refFlat mappings of the refSeq genes don't have good concordance with the protein sequences from GenBank.  A value of 0 indicates a frame-check could be performed, but no in-frame contig was found.  1 means in-frame.  2 and 3 are very similar, they both indicate that in the geneB portion of the contig the transcription start site for geneB is present; the differences are in how the site is identified and I'm not sure how much practical difference it makes.  Codes 2 and 3 will only be reported if an in-frame call (value 1) is not found, this is more of a rescue/salvage annotation.

**Reply by ahdee** [reply]
@adthrasher @mnedmonson  thank you this is perfect.
